identity/app/views/advertsManager.scala.html

```diff
@@ -6,17 +6,17 @@
 
 <div class="l-side-margins">
 
-<div class="identity-ad-prefs-area identity-ad-prefs-area--header">
-    <div class="gs-container">
-        <header class="identity-ad-prefs-area__section">
-            <h1 class="identity-title">Cookies and advertising settings</h1>
-        </header>
-    </div>
-</div>
-
 <div class="identity-ad-prefs-area">
 
     <div class="gs-container">
+         <section class="identity-ad-prefs-area__section">
+            <h1 class="identity-title">Your Privacy</h1>
+            <h3 class="identity-title identity-title--small">We use cookies</h3>
```

Comment:
Nit: can this be a `h2` for better page semantics?

Reply:
Will do!